docs/development/index.md

```diff
@@ -138,7 +138,7 @@ Documentation uses MkDocs Material:
 pip install mkdocs-material
 
 # Serve docs locally
-mkdocs serve
+mkdocs serve -f docs/.config/mkdocs.yml
 
 # Build docs
 mkdocs build
```

Comment:
do we need to update here as well? 

Or we can consider using `make docs-serve`, `make docs-build` from Makefile?